xmppserver/src/main/java/org/jivesoftware/openfire/commands/admin/GetListActiveUsers.java

```diff
@@ -48,22 +51,24 @@ protected void addStageInformation(SessionData data, Element command) {
 
         field = form.addField();
         field.setType(FormField.Type.list_single);
-        field.setLabel("Maximum number of items to show");
+        field.setLabel(LocaleUtils.getLocalizedString("commands.global.operation.pagination.max_items", preferredLocale));
         field.setVariable("max_items");
         field.addOption("25", "25");
         field.addOption("50", "50");
         field.addOption("75", "75");
         field.addOption("100", "100");
         field.addOption("150", "150");
         field.addOption("200", "200");
-        field.addOption("None", "none");
+        field.addOption(LocaleUtils.getLocalizedString("commands.global.operation.pagination.none", preferredLocale), "none");
 
         // Add the form to the command
         command.add(form.getElement());
     }
 
     @Override
-    public void execute(SessionData data, Element command) {
+    public void execute(@Nonnull final SessionData data, Element command) {
+        final Locale preferredLocale = SessionManager.getInstance().getLocaleForSession(data.getOwner());
+
         String max_items = data.getData().get("max_items").get(0);
```

Comment:
There's an attempt in this block to make max_items optional, as per https://xmpp.org/extensions/xep-0133.html#example-71 

If it's not supplied, this line gives an NPE

```2024.01.09 21:52:13.311 ERROR [nioEventLoopGroup-3-1]: org.jivesoftware.openfire.handler.IQHandler - Internal server error
java.lang.NullPointerException: null
    at org.jivesoftware.openfire.commands.admin.GetListActiveUsers.execute(GetListActiveUsers.java:72) ~[xmppserver-4.8.0-SNAPSHOT.jar:4.8.0-SNAPSHOT]
    at org.jivesoftware.openfire.commands.AdHocCommandManager.process(AdHocCommandManager.java:286) ~[xmppserver-4.8.0-SNAPSHOT.jar:4.8.0-SNAPSHOT]
    at org.jivesoftware.openfire.commands.AdHocCommandHandler.handleIQ(AdHocCommandHandler.java:80) ~[xmppserver-4.8.0-SNAPSHOT.jar:4.8.0-SNAPSHOT]```

Packet capture:

```
    <packet xmlns="http://www.jivesoftware.org" streamID="2cgqol3kef" status="auth"
        timestamp="Jan 09, 2024 09:52:13:311 PM">
        <iq xmlns="" to="admin@example.org" id="LJHSU-14" type="set"
            from="admin@example.org/4a7b1d90-c978-45c9-980d-4236e3a93ac6">
            <command xmlns="http://jabber.org/protocol/commands"
                node="http://jabber.org/protocol/admin#get-active-users" sessionid="5f2HSYXr68UZxbf" action="complete">
                <x xmlns="jabber:x:data" type="submit">
                    <field var="FORM_TYPE">
                        <value>http://jabber.org/protocol/admin</value>
                    </field>
                </x>
            </command>
        </iq>
    </packet>
    <packet xmlns="http://www.jivesoftware.org" streamID="2cgqol3kef" status="auth"
        timestamp="Jan 09, 2024 09:52:13:314 PM">
        <iq xmlns="" type="error" id="LJHSU-14" from="admin@example.org"
            to="admin@example.org/4a7b1d90-c978-45c9-980d-4236e3a93ac6">
            <command xmlns="http://jabber.org/protocol/commands"
                node="http://jabber.org/protocol/admin#get-active-users" sessionid="5f2HSYXr68UZxbf" action="complete">
                <x xmlns="jabber:x:data" type="submit">
                    <field var="FORM_TYPE">
                        <value>http://jabber.org/protocol/admin</value>
                    </field>
                </x>
            </command>
            <error code="500" type="wait"><internal-server-error xmlns="urn:ietf:params:xml:ns:xmpp-stanzas" /></error>
        </iq>
    </packet>
```

Reply:
I have now added an explicit check that should prevent this.